app/frontend/styles/_filter.scss

```
@@ -29,3 +29,9 @@
 .moj-filter__options {
   box-shadow: none;
 }
+
+.app-filter {
+  @include govuk-media-query($until: filter_hide) {
+    display: none;
+  }
+}
```

Here is how to rewrite this "mobile-first", which is functionally identical, but easier to add to in the future when designing for phones:

```suggestion
.app-filter {
  display: none;

  @include govuk-media-query($from: large) {
    display: block;
  }
}
```

---

👍